### PR TITLE
wr-232: Add ability for authors to edit and delete their articles directly from the article viewing page

### PR DESCRIPTION
**Pull request by PablloZz**
image
image
image
image

 Add ability to edit my articles from the "Feed" tab. Was implemented in @AnastasiaR2 PR.
 Add ability to edit my articles from the article viewing page.
 Add ability to delete my articles from the "Feed" tab. Was implemented in @AnastasiaR2 PR.
 Add ability to delete my articles from the article viewing page.

### CODE REVIEW

File: frontend/src/pages/article/components/article-view/styles.module.scss [hunk 1]
```diff
@@ -39,10 +39,14 @@
 
 .iconButton {
   display: flex;
+  flex: 1 1 auto;
   justify-content: center;
   align-items: center;
-  width: 54px;
-  height: 54px;
+  min-width: 30px;
```
Comment (artemmatiushenko1) [inline frontend/src/pages/article/components/article-view/styles.module.scss]: I guess we can just remove `top: 100px;` from `buttonsWrapper`, so each button doesn't shrink in size. Currently the icons look quite large

File: frontend/src/libs/constants/previous-page-index.ts
Comment (andanf-e) [inline frontend/src/libs/constants/previous-page-index.ts:1]: Should be `previous-page-index.constant`

File: frontend/src/pages/article/components/article-view/article-view.tsx [hunk 2]
```diff
@@ -53,6 +76,27 @@ const ArticleView: React.FC<Properties> = ({
...
+                <IconButton
+                  iconName="trashBin"
+                  className={styles.iconButton}
+                  iconClassName={styles.icon}
+                  onClick={handleDeleteArticle}
+                />
+                <RouterLink
+                  to={AppRoute.EDIT_ARTICLE.replace(':id', id as string)}
+                  state={article}
+                >
```
Comment (andanf-e) [inline frontend/src/pages/article/components/article-view/article-view.tsx]: Use common `Link` component,
And please change it on article-card component,
I missed it

**Review by andanf-e — CHANGES_REQUESTED** [review]
And fix issues with package-lock

**Review by andanf-e — APPROVED** [review]

**Pull request merged**